src/main/webapp/scripts/jest-example.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * This function provides a simple example of how to use the Jest module.
+ * It has nothing to do with the functionality of the overall project.
+ */
+function sum(a, b) {
+  return a + b;
+}
+
```

Comment:
I think at this point you could delete this function (and the ones associated with it)